.env.default

```diff
@@ -0,0 +1,11 @@
+CORS_ORIGIN=^http:\/\/localhost:[0-9]{1,10}$
+CORS_METHODS=*
+
+HTTP_SERVER_PORT=5001
+HTTP_SERVER_HOST=0.0.0.0
+AWS_CLUSTER_NAME=dev-main
```

Comment:
why dev-main?

Reply:
to just pick a name, as an example, I changed to blank but it's not that important IMO